spec/integration/cart_view_spec.rb

```diff
@@ -5,6 +5,9 @@
 
   before(:each) do
     visit cart_path
+    user = FactoryGirl.build(:user)
+    require 'pry'; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.